recipe/meta.yaml

```
@@ -1,34 +1,40 @@
-{% set name = "Pyphen" %}
+{% set name = "pyphen" %}
 {% set version = "0.9.5" %}
-{% set sha256sum = "3b633a50873156d777e1f1075ba4d8e96a6ad0a3ca42aa3ea9a6259f93f18921" %}
+
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256sum }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/Pyphen-{{ version }}.tar.gz
+  sha256: 3b633a50873156d777e1f1075ba4d8e96a6ad0a3ca42aa3ea9a6259f93f18921
 
 build:
   noarch: python
-  number: 0
-  script: python -m pip install --no-deps --ignore-installed .
+  number: 1
+  script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
-    - pip
-    - python
-    - setuptools
+    - pip >=21.0
+    - python >=3.6
 
   run:
-    - python
+    - python >=3.6
+
+test:
+  imports:
+    - pyphen
+  commands:
+    - pip check
+  requires:
+    - pip
 
 about:
   home: http://pyphen.org/
-  license: GPL 2.0+/LGPL 2.1+/MPL 1.1
-  license_file: COPYING
+  license: GPL-2.0-or-later, LGPL-2.1-or-later, MPL-1.1
```

```suggestion
  license: GPL-2.0-or-later AND LGPL-2.1-or-later AND MPL-1.1
```
If it's an OR, you can replace AND with OR above

---

Gotcha, I will make that change. Thanks! @isuruf